Makefile.in

```diff
@@ -160,6 +176,19 @@ libssh.a: $(LIBSSH_OBJS)
 	$(AR) rv $@ $(LIBSSH_OBJS)
 	$(RANLIB) $@
 
+ironsftp$(EXEEXT): $(LIBCOMPAT) sftp.o sftp-client.o sftp-common.o sftp-glob.o iron-gpg.o libssh.a
+	$(LD) -o $@ sftp.o sftp-client.o sftp-common.o sftp-glob.o iron-gpg.o $(LDFLAGS) -lssh -lopenbsd-compat $(LIBS)
+
+ironsftp-server$(EXEEXT): $(LIBCOMPAT) sftp-server.o sftp-server-main.o sftp-common.o libssh.a
+	$(LD) -o $@ sftp-server.o sftp-server-main.o sftp-common.o $(LDFLAGS) -lssh -lopenbsd-compat $(LIBS)
+
+ironscp$(EXEEXT):  $(LIBCOMPAT) scp.o libssh.a
+	$(LD) -o $@ scp.o $(LDFLAGS) -lssh -lopenbsd-compat $(LIBS)
+
+irongpg$(EXEEXT): iron-gpg-test.o sftp-common.o libssh.a $(LIBCOMPAT)
```

Comment:
What is irongpg?  Is that something I can use to inspect files?  I just tried `make irongpg` but the build failed.